app/components/point.jsx

```diff
@@ -0,0 +1,175 @@
+// https://github.com/EnCiv/civil-pursuit/issues/23
+
+'use strict'
+import React from 'react'
+import cx from 'classnames'
+import { createUseStyles } from 'react-jss'
+
+function Point(props) {
+  const { subject, description, vState, children, className, ...otherProps } = props
+
+  const classes = useStylesFromThemeFunction()
+
+  const borderClass = classes[`${vState}Border`]
+  const subjectClass = classes[`${vState}Subject`]
+  const descriptionClass = classes[`${vState}Description`]
+
+  const childrenWithProps = React.Children.map(children?.props?.children ?? children, child => {
+    return React.cloneElement(child, {
+      className: className,
```

Comment:
The new, new way to do this is to go back to having the isHovered state, create a class for what you want the children to look like when this component is hovered, and use this to apply that class to the children.
```
className: cx({ [classes.hovered]: isHovered } , className)
```
Then the children won't have to be have custom handling for being hovered under the Point compoent.

The idea being  the Point component can have generic children, not written with anything specific to the point component.

Lets discuss in the meeting tomorrow.